### PR TITLE
Adds tests: reviews/destroy_spec, reviews/edit_spec, models/review_spec

### PR DESCRIPTION
**Pull request by fentontaylor**
New feature tests:
- spec/features/reviews/edit_spec.rb (not passing)
- spec/features/reviews/destroy_spec.rb (not passing)

New model test:
- spec/models/review_spec.rb (passing)

### CODE REVIEW

File: spec/features/reviews/destroy_spec.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe 'User visits the item show page' do
+  describe 'They can click a link and fill in form to edit review' do
+    it 'Edits the original review' do
```
Comment (alect47) [inline spec/features/reviews/destroy_spec.rb]: Just in terms or what the test is named
- is it editing the original review or deleting it?  or are the two a combined action?

File: spec/features/reviews/destroy_spec.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      visit "/items/#{chain.id}"
+
+      within "#review-#{review.id}" do
+        expect(page).to have_link('Delete')
+        click_link 'Delete'
+      end
+
+      expect(current_path).to eq("/items/#{chain.id}")
+
+      expect(page).to_not have_css("#item-#{chain.id}")
```
Comment (alect47) [inline spec/features/reviews/destroy_spec.rb]: Should we add more tests here to make sure that it doesn't contain any of the review attributes?

**Comment by alect47** [comment]
Left a couple comments in the code, let me know what you think

File: spec/features/reviews/destroy_spec.rb (hunk 1 above)
Comment (fentontaylor) [reply spec/features/reviews/destroy_spec.rb]: oops that actually should have been changed, i copied and pasted most of the test from the previous one

**Pull request merged**